server/service/scripts.go

```diff
@@ -746,6 +746,114 @@ func (svc *Service) GetScript(ctx context.Context, scriptID uint, withContent bo
 	return script, content, nil
 }
 
+////////////////////////////////////////////////////////////////////////////////
+// Update Script Contents
+////////////////////////////////////////////////////////////////////////////////
+
+type updateScriptRequest struct {
+	Script   *multipart.FileHeader
+	ScriptID uint
```

Comment:
I think you need to add `url:"script_id"` here

Reply:
you didn't have to 'cuz you were getting it from the POST 😉 .  But it's probably why your test started failing.

Reply:
@sgress454 Aaah... okay so apparently `DecodeRequest` doesn't handle URL parameters for you, but `DecodeBody` does. Sneaky! https://github.com/fleetdm/fleet/blob/8b23ed5262277fb0a7916ffaa25f6aed3b3d65e3/server/service/endpoint_utils.go#L106-L112

Reply:
Fixede 😎